experimental/cdf_asset_source_model/transformations/sync-asset_hierarchy_cdf_asset_source_model.sql

```diff
@@ -0,0 +1,48 @@
+-- Root Asset
+-- The asset must be set up in hierarchical order as the container for the parent asset requires the
+-- parent asset to be created first.
+select
```

Comment:
Would be great with a simple description of what the logic behind the select is.